libs/api/src/lib/graphql/subscription/subscription.private-mutation.ts

```diff
@@ -123,11 +152,6 @@ export const updateAdminSubscription = async (
 
   if (!updatedSubscription) throw new NotFound('subscription', id)
 
-  // cancel open invoices if subscription is deactivated
-  if (deactivation !== null) {
```

Comment:
why do you delete that? sure? wasn't that a former fix?

Reply:
Gibt doch jetzt einen eigenen endpoint für die deaktivierung oder?